axes/signals.py

```diff
@@ -148,7 +148,9 @@ def log_user_logged_in(sender, request, user, **kwargs):  # pylint: disable=unus
         )
 
     if settings.AXES_RESET_ON_SUCCESS:
-        count = reset_user_attempts(request)
+        # Create credentials dictionary from the username field
+        credentials = {settings.AXES_USERNAME_FORM_FIELD: username}
```

Comment:
why overwrite credentials?

Reply:
I'm not overwriting them here.
Did you maybe add that comment to the wrong line?
log_user_logged_in doesn't receive the credentials object from the authentication backend, but is only passed a user object. (which is not identical to the credentials dictionary)

Reply:
you're right i was looking at the wrong line